fetch/fetch.go

```diff
@@ -554,12 +556,12 @@ func (f *Fetch) sendBatch(requests []requestMessage) {
 
 // handleHashError is called when an error occurred processing batches of the following hashes
 func (f *Fetch) handleHashError(batchHash types.Hash32, err error) {
-	f.log.Debug("cannot fetch message %v err %v", batchHash.Hex(), err)
+	f.log.Debug("cannot fetch message %v err %v", batchHash.ShortString(), err)
```

Comment:
can refactor this to use formatted logs (put `batchHash` and `err` as parameters

Reply:
done for the whole file.